coderd/workspace_apps_internal_test.go

```diff
@@ -0,0 +1,100 @@
+package coderd
```

Comment:
This file name should be `workspaceapps_internal_test.go` to be consistent with `workspaceapps.go`